.github/workflows/main.yml

```diff
@@ -0,0 +1,11 @@
+name: A workflow for my Hello World file
+on: push
+jobs:
+  build:
+    name: Hello world action
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: ./action-a
+        with:
+          MY_NAME: "Mona"
```

Comment:
This is your custom action.  As you found out with `industrial`, you often times don't need your own Action since Actions are built to be shared.  This is good practice to understand how they work to take the mystery out of it.